studybuddies/src/app/components/student/class-list-student/class-list-student.component.ts

```diff
@@ -1,4 +1,5 @@
 import { Component, OnInit } from '@angular/core';
+import { SalasService } from 'src/app/services/salas.service';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7203036)
```suggestion
import { SalasService } from "src/app/services/salas.service";
```